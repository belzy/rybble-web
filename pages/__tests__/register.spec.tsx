import { render } from '@testing-library/react';
import RegisterPage from '../register';
import '@testing-library/jest-dom';

describe('pages/register', () => {

  it('should render', () => {

    const { container } = render(<RegisterPage />);

    expect(container).toMatchSnapshot();

  });

});