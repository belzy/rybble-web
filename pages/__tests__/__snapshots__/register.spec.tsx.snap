// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`pages/register should render 1`] = `
<div>
  <div
    class="css-qy22cp"
  >
    <div
      class="css-1wb80v"
    >
      <div
        class="carousel slide"
      >
        <div
          class="carousel-indicators"
        >
          Testing
        </div>
        <div
          class="carousel-inner"
        >
          Testing
        </div>
        <a
          class="carousel-control-prev"
          href="#"
          role="button"
          tabindex="0"
        >
          <span
            aria-hidden="true"
            class="carousel-control-prev-icon"
          />
          <span
            class="visually-hidden"
          >
            Previous
          </span>
        </a>
        <a
          class="carousel-control-next"
          href="#"
          role="button"
          tabindex="0"
        >
          <span
            aria-hidden="true"
            class="carousel-control-next-icon"
          />
          <span
            class="visually-hidden"
          >
            Next
          </span>
        </a>
      </div>
    </div>
    <div
      class="css-b95f0i"
    />
  </div>
</div>
`;
